gnomad/gnomadv4_extract_test.py

```diff
@@ -0,0 +1,50 @@
+# LT 23/08/2022
+
+## Extract a small subset of 10 transcripts from gnomADv4
+## annotate with frequencies
+## export the rows as a hail.Table
+
+
+# target transcripts for panel of initial 10 genes selected by the structural constraint project
+# TODO should be in an external file
```

Comment:
Maybe just a `json` file, which is super easy to read (`json.load`)?

Reply:
implemented